### ISSUE TITLE
Cell display sometimes doesn't display voltages

### ISSUE THREAD

**Issue by mucek** [vidklopcic/eCarMonitoringApp#11]
In some (rare) cases, cell voltages aren't displayed after some display sweeping left/right. Only temperature is displayed. A full app restart must be done in order to get details back.

**Comment by mucek** [comment]
Rechecked: Issue appears, after sliding to right screen (empty)

**Comment by vidklopcic** [comment]
Do the values disappear every time you swipe to the right screen?

**Comment by mucek** [comment]
No. First time when swiping right valves are lost until service restart.
